### ISSUE TITLE
Plugin 'mobi.hsz.idea.gitignore' failed to initialize and will be disabled.  Please restart Android Studio.

### ISSUE THREAD

**Issue by yizeliang** [hsz/idea-gitignore#323]
```
Plugin 'mobi.hsz.idea.gitignore' failed to initialize and will be disabled.  Please restart Android Studio.

java.lang.NullPointerException
    at mobi.hsz.idea.gitignore.IgnoreApplicationComponent.initComponent(IgnoreApplicationComponent.java:59)
    at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:522)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.createComponents(ComponentManagerImpl.java:123)
    at com.intellij.openapi.application.impl.ApplicationImpl.access$801(ApplicationImpl.java:92)
    at com.intellij.openapi.application.impl.ApplicationImpl$12.run(ApplicationImpl.java:500)
    at com.intellij.openapi.progress.impl.CoreProgressManager$2.run(CoreProgressManager.java:142)
    at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:446)
    at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:392)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
    at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:127)
    at com.intellij.openapi.application.impl.ApplicationImpl.createComponents(ApplicationImpl.java:516)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:107)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:458)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:450)
    at com.intellij.idea.IdeaApplication.run(IdeaApplication.java:197)
    at com.intellij.idea.MainImpl$1$1$1.run(MainImpl.java:52)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
    at java.awt.EventQueue.access$500(EventQueue.java:97)
    at java.awt.EventQueue$3.run(EventQueue.java:709)
    at java.awt.EventQueue$3.run(EventQueue.java:703)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:366)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by akshaybabloo** [comment]
I get the same error #324

**Comment by yizeliang** [comment]
@akshaybabloo 

Android 2.2.3 , down: https://github.com/hsz/idea-gitignore/releases/tag/v1.7.4

select install from disk

don't update it

**Comment by minhphuc429** [comment]
@yizeliang it work, thanks!

**Comment by hsz** [comment]
Thank you for the report. It has been already reported and fixed with #315
